src/mixins/ContentMixin.js

```diff
@@ -310,6 +310,12 @@ export default {
     requestSynchronisedEvent: function () {
       return;
     },
+    flatmapReadyCall: function (flatmap) {
```

Comment:
Can this be moved to the Flatmap.vue? I tried not to expose method here unless it is used by more than one module.

Reply:
yeah can do :)